begoneads/begoneads.desktop

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env xdg-open
+[Desktop Entry]
+Type=Application
+Name=Begoneads
+Exec=xterm -e sudo python $(find $(pip show begoneads | grep Location | awk '{print $2}') -name "begoneadsGUI.py")
```

Comment:
What happens when the user does not have `xterm` installed?

The `find` construction also doesn't look very durable. Is there a better solution to this?

Reply:
idk about this. you need something to run the script in. xterm should be installed in most linux operating systems. this is a convenience you can also run the begoneadsGUI script and it will work